lighthouse/blueprints/plates.py

```diff
@@ -52,11 +57,20 @@ def create_plate_from_barcode() -> Tuple[Dict[str, Any], int]:
         if response.ok:
             response_json = {
                 "data": {
-                    "plate_barcode": samples[0]["plate_barcode"],
+                    "plate_barcode": samples[0][FIELD_PLATE_BARCODE],
                     "centre": centre_prefix,
                     "number_of_positives": len(samples),
                 }
             }
+
+            try:
+                update_mlwh_with_cog_uk_ids(samples)
```

Comment:
send_to_ss(body) in line 55 will create the plate and samples, but the samples creation in the warehouse is asynchronous because is happening by sending a message to the queues. The new lighthouse_sample table in the warehouse doesn't seem to have foreign keys we think (from the definition of the test schema setup_test_db.py) so it can create entries independently from the samples table, without having the sample yet in the database. Is that the intention?

Reply:
Just thinking, we already have a process that writes in the warehouse (unified_warehouse) and updates samples information. What was the reason why the update on cog-uk-ids was not happening in that project unified_warehouse? I kind of remember that the code was very coupled with the tables it was updating, was this the reason?

Reply:
This seems like it might be a problem! Not quite sure what you're suggesting for the alternative - would be good to have a chat about it - but we're going to try to crack on with testing the crawler stuff at least in UAT first. 

Thanks